assets/app/views/browse/_build-details.html

```diff
@@ -38,7 +31,7 @@
           </span>
         </dd>
       </dl>
-      <h3>Configuration <span class="small" ng-if="buildConfigName">created from <a href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfigName}}">{{buildConfigName}}</a></span></h3>        
+      <h3>Configuration <span class="small" ng-if="buildConfigName">created from <a href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfigName}}">{{buildConfigName}}</a></span></h3>
```

Comment:
or wait and rebase after your other one merges, but who knows when that will actually go through :)